### ISSUE TITLE
component "demo-graph" not found

### ISSUE THREAD

**Issue by ychakiris** [phantomics/seed#2]
I followed the directions for loading the program into common lisp (SBCL) using quicklisp. Unfortunately ASDF could not find "demo-graph".

Did you remove or refactor this out of the code?

**Comment by aykaramba** [comment]
Agreed, just managed to load up the revival brand and that folder is missing for me as well.
